doc/cmp.txt

```diff
@@ -366,6 +366,16 @@ enabled~
   `boolean | fun(): boolean`
   Toggles the plugin on and off.
 
+                                                *cmp-config.performance.debounce*
+performance.debounce~
+  `number`
+  Sets debounce time
```

Comment:
I think we should add more obvious description for this.

Reply:
Hm... the `debounce` is hard to explain.

If the user press `console` and it's invoking 10 number of sources, the nvim-cmp will retrieve completion results 10 times.

Typically, the each responses would come back from sources in small milliseconds.
I think it should be merged as one filtering process. The debounce_times is used for this.

Reply:
So debounce is basically how long cmp waits to retrieve completion results from sources before displaying.